src/ResizeSensor/ResizeSensor.ts

```diff
@@ -1,28 +1,41 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import _debounce from 'lodash/debounce';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import _debounce = require('lodash/debounce');
 import { props, t, ReactChildren } from '../utils';
-import _ResizeSensor from 'css-element-queries/src/ResizeSensor';
+import { ResizeSensor as _ResizeSensor } from 'css-element-queries';
```

Comment:
not strictly necessary, but I noticed that `ResizeSensor` is exported from the `index.js`